teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization)
+    @specialization = specialization
+    super
+  end
```

Comment:
- [ ] The same goes here in the Teacher class. Please try to refer to the review I have given above and the suggestion I have given you in order to correct the initialize method here in the Teacher class. In general, It will look almost like the one in Student class 😉

Reply:
Thank you @BertrandConxy, updated the project according to your guidance.